src/marqo_commons/model_registry/model_properties_data/open_clip_properties.py

```diff
@@ -368,6 +369,7 @@ def get_all_model_properties_objects(cls) -> Dict[str, "OpenClipModelProperties"
                 dimensions=1152,
                 notes="open_clip model: ViT-SO400M-14-SigLIP-384/webli",
                 pretrained="webli",
+                imageProcessor="SigLIP",
```

Comment:
size=384 for this model. check https://huggingface.co/timm/ViT-SO400M-14-SigLIP-384/blob/ac16108d567c4389e6cd2b11c9b8585f7474435b/open_clip_config.json#L7

Reply:
Check other models.